src/iframe-helper.js

```diff
@@ -416,8 +416,12 @@ export class SubscriptionApi {
    * @param {boolean} is3p set to true if the iframe is 3p.
    * @param {function(!JsonObject, !Window, string)} requestCallback Callback
    *     invoked whenever a new window subscribes.
```

Comment:
Looks like we are safe. As a test, I moved the remote frame to a server on a different port (thus different origin). I added this code to the remote frame, and it did throw exceptions both times:
```
        try {
          console.log(window.top.location.href);
        } catch (e) {
          console.warn(e.message);
        }
        try {
          console.log(window.top.document.childElementCount);
        } catch (e) {
          console.warn(e.message);
        }
```